content/200-concepts/100-components/02-prisma-client/250-opentelemetry-tracing.mdx

```diff
@@ -146,13 +146,61 @@ registerInstrumentations({
 provider.register()
 ```
 
-OpenTelemetry is highly configurable. You can customize the resource attributes, what components gets instrumented, how spans are processed and where spans are sent.
+OpenTelemetry is highly configurable. You can customize the resource attributes, what components gets instrumented, how spans are processed, and where spans are sent.
 
 You can find a complete example that includes metrics in [this sample application](https://github.com/garrensmith/prisma-metrics-sample).
 
-## Advanced tracing tasks
+## Tracing how-tos
 
-This section gives an introduction to more advanced tracing tasks that you may need to perform.
+### Send tracing output to the console
```

Comment:
I would put this _after_ the normal visualization with Jaeger - that one is the much more common use case, while the logging to console is more a debugging and testing thing.

Reply:
Makes sense - OK, I'll move it.

Reply:
This is done, so I'm closing the conversation